make-release.sh

```diff
@@ -6,24 +6,45 @@
 # set to 1 to actually trigger changes in the release branch
 TRIGGER_RELEASE=0 
 NOCOMMIT=0
+USE_TMP_DIR=0
+REPO=git@github.com:eclipse/che-devfile-registry
 
 while [[ "$#" -gt 0 ]]; do
   case $1 in
     '-t'|'--trigger-release') TRIGGER_RELEASE=1; NOCOMMIT=0; shift 0;;
-    '-r'|'--repo') REPO="$2"; shift 1;;
     '-v'|'--version') VERSION="$2"; shift 1;;
+    '-tmp'|'--use-tmp-dir')USE_TMP_DIR=1; shift 0;;
     '-n'|'--no-commit') NOCOMMIT=1; TRIGGER_RELEASE=0; shift 0;;
   esac
   shift 1
 done
 
 usage ()
 {
-  echo "Usage: $0 --repo [GIT REPO TO EDIT] --version [VERSION TO RELEASE] [--trigger-release]"
-  echo "Example: $0 --repo git@github.com:eclipse/che-subproject --version 7.7.0 --trigger-release"; echo
+  echo "Usage: $0  --version [VERSION TO RELEASE] [--trigger-release]"
+  echo "Example: $0 --version 7.7.0 --trigger-release"; echo
```

Comment:
nitpick/personal preference: when I update scripts and the examples change, I update the example to the current version of Che... so instead of an ancient 7.7.0 example, we ought to have 7.27.0 here as that's when this will be used first.